File: hw/codena/u1/hw06Files/JavaScript/model/Student.js
class Student {

    constructor(id, name, course) {
        this.id = id;
        this.name = name;
        this.course = course;
    }

    toString() {
        return `Student{
id=${this.id},
name=${this.name},
course=${this.course}
}`;
    }
}

export default Student;

File: my-fresh-copy/hw/codena/u1/hw06Files/Python/model/student.py
class Student:

    def __init__(self, id, name, course):
        self.id = id
        self.name = name
        self.course = course

    def to_dict(self):
        return {
            "id": self.id,
            "name": self.name,
            "course": self.course
        }

    def __str__(self):
        return (f"Student("
                f"id={self.id}, "
                f"name={self.name}, "
                f"course={self.course})")